relay-server/src/services/buffer/envelope_store/sqlite.rs

```diff
@@ -476,25 +484,82 @@ impl SqliteEnvelopeStore {
     }
 }
 
+fn pack_envelopes(envelopes: Vec<DatabaseEnvelope>) -> Box<[u8]> {
+    let mut packed = vec![];
+    for envelope in envelopes {
+        packed.extend_from_slice(&envelope.received_at.to_le_bytes());
+        packed.extend_from_slice(&(envelope.encoded_envelope.len() as u32).to_le_bytes());
+        packed.extend_from_slice(&envelope.encoded_envelope);
```

Comment:
My first implementation used MsgPack, but this homegrown format turned out to be much faster.